app/src/main/java/org/jellyfin/mobile/AppPreferences.kt

```diff
@@ -29,6 +30,14 @@ class AppPreferences(context: Context) {
             }
         }
 
+    var currentUserUuid: UUID?
+        get() = sharedPreferences.getString(Constants.PREF_USER_UUID, null)?.let(UUID::fromString)
+        set(value) {
+            sharedPreferences.edit {
+                if (value != null) putString(Constants.PREF_USER_UUID, value.toString()) else remove(Constants.PREF_USER_UUID)
+            }
+        }
```

Comment:
This part is basically a clone of the above `currentUserId` but uses the server given `UUID` instead of the locally created `Long`. We might want to merge those two.